src/init-ceph.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
i mean, user installs a POSIX compliant shell by default, but it could be dash or any other shell implementations. we'd better not force user to install bash even it's ubiquitous. and i don't think, bash is not equivalent to GNU/linux.